domain-models-runtime/src/main/java/org/folio/rest/persist/PostgresClient.java

```diff
@@ -3281,6 +3281,52 @@ void getSQLConnection(int queryTimeout, Handler<AsyncResult<SQLConnection>> hand
     });
   }
 
+  /**
+   * Execute the given function within a transaction.
+   * <p>Similar to {@link PgPool#withTransaction(Function)}
+   * <ul>
+   *   <li>The connection is automatically closed in all cases when the function exits.</li>
+   *   <li>The transaction is automatically committed if the function returns a succeeded Future.
+   *   The transaction is automatically roll-backed if the function returns a failed Future or throws a Throwable.</li>
+   *   <li>The method returns a succeeded Future if the commit is successful, otherwise a failed Future.</li>
+   * </ul>
+   *
+   * @param function code to execute
+   * @param <T> type
+   * @return result handler
```

Comment:
Should be removed as it doesn't give any further information.

Reply:
Personally, I prefer to have it. But I'll remove.